userge/plugins/admin/gban.py

```diff
@@ -0,0 +1,224 @@
+#(｡ŏ_ŏ) GBan module for userge 
+
+import requests
+import logging
+import asyncio
+from userge import userge, Message, Config, get_collection, Filters
+
+GBAN_USER_BASE = get_collection("GBAN_USER")
+gban = GBAN_USER_BASE
+GBAN_LOG = Config.GBAN_LOG_CHANNEL
```

Comment:
you can simply use userge.getCLogger(__name__)

```suggestion
GBAN_LOG = userge.getCLogger(__name__)
```